cli/dcoscli/marathon/main.py

```diff
@@ -656,7 +656,7 @@ def _clean_up_resource_definition(properties):
         if v:
             if k in ["apps", "groups"]:
                 clean_properties[k] = [_clean_up_resource_definition(v[0])]
-            elif not k.startswith("task"):
+            elif not k.startswith("task") and k != "lastTaskFailure":
```

Comment:
Let's add a comment as to why we are doing this. A comment for both `...startswith("task")` and `... "lastTaskFaillure"` I think I have ask the same question more than once already.